lib/report.js

```diff
@@ -112,7 +100,17 @@ class ConciseReporter extends Reporter {
   listFailure(test, res, message) {
     if (res.stack) res.stack = renderStack(res.stack);
     this.printAssertErrorSeparator();
-    console.log(list(res, message, ['success']) + '\n');
+    let s = message;
+    for (const key of ['type', 'message', 'expected', 'actual', 'stack']) {
```

Comment:
Wouldn't it be better to iterate `Object.keys(res)` here and just add a check
```javascript
    if (key === 'success') continue;
```
inside a loop?

Reply:
Nope, I needed specific key order. It was simple iteration before which resulted in unsighty output.

Reply:
And what about replacing this for loop with `forEach`?

Reply:
Okay, I'm fine either way.